src/response.ts

```diff
@@ -60,8 +58,8 @@ function createResponseComposition(
     )
 
     if (transformers.length > 0) {
-      return compose(...postProcessTransformers.concat(transformers))(
-        resolvedResponse,
+      return JSONBodyResponseTransformer(
```

Comment:
Is this what you had in mind? I find it confusing that the `createResponseComposition` returns a `ResponseFunction`. While `compose(...t)(response)` to my understanding doesn't return a `ResponseFunction` but just the `response` which I see also is being passed to the `Object.assign` below?